src/commands/stats.ts

```diff
@@ -13,6 +17,13 @@ export const stats: CommandDef = {
     try {
       const uptime = getUpTime(client);
 
+      // If run locally, it will get the commit hash using a different approach
+      const versionHash = process.env.SOURCE_VERSION;
+      const commitHash =
+        versionHash == undefined
+          ? info.abbreviatedSha
+          : versionHash.slice(0, 10);
```

Comment:
I guess the `SOURCE_VERSION` will be found when running locally, otherwise the `info.abrviatedSha` will be loaded? Isn't `info` always available? Or does this not work in some environments/situations?

this could probably could be done with just:
```ts
const commitHash = process.env.SOURCE_VERSION || info.abbreviatedSha;
```

Reply:
`SOURCE_VERSION` will be found when it's deployed in heroku. But, I will fix this. I just never noticed that if the value if `undefined` or `null` it will go to the fallback.